### ISSUE TITLE
Remove Font Awesome icon overrides in USWDS components

### ISSUE THREAD

**Issue by jamigibbs** [department-of-veterans-affairs/vets-design-system-documentation#2577]
## Description
We would like to start using the USWDS icons that they provide in their component modules instead of Font Awesome. Reference this [Slack thread](https://dsva.slack.com/archives/C01DBGX4P45/p1710200196112369).

We have done this for `va-accordion` already and we need to check the other components to make sure there aren't overrides there too:
- https://github.com/department-of-veterans-affairs/component-library/pull/1083

## Tasks
- [ ] Audit the web components to find where Font Awesome is used
- [ ] Remove Font Awesome override(s) from USWDS components to allow USWDS modules to load icons instead
- [ ] Replace other Font Awesome icons with va-icon
- [ ] Update Figma files (maybe a separate issue for designers)

## Acceptance Criteria
- [ ] All Font Awesome icons in USWDS components are displaying USWDS icons instead

**Comment by caw310** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Platform-Design-System-5f8de67192551b0012ebb802/issues/department-of-veterans-affairs/vets-design-system-documentation/2577?planning-poker) with Zenhub @Andrew565 @ataker @harshil1793 @it-harrison @jamigibbs @micahchiang @nickjg231 @powellkerry @rmessina1010 @rsmithadhoc

**Comment by Andrew565** [comment]
Also need to bring in a few new icons to the va-icon component, including adding those new icons to Storybook.

**Comment by Andrew565** [comment]
@caw310 This ticket will carry-over due to needing to fix a couple design issues, so there likely won't be enough time to get it through the review process today.

**Comment by caw310** [comment]
This ticket is blocked by 2680 so will likely have to carry over.
